src/locales/en-US/common.ftl

```diff
@@ -2,8 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+product-line = Firefox
 product-title = Lockbox
 product-tagline = The simple way to store, retrieve and manage website login info
+product-fulltitle = { product-line } { product-title }
```

Comment:
We should check with the content / branding folks about localizing the full title, either now or in a followup bug (and before we turn our localizers loose on this project). Product names are often [not translated at all](https://github.com/mozilla-services/screenshots/blob/master/locales/en-US/server.ftl#L69), and if that's the decision here, we may want to add a note above each of product-line, product-title, and product-fulltitle, asking localizers not to translate these, like:

```
# Note: please don't translate Firefox
product-line = Firefox
```

It's probably safe to assume these won't be translated, and add the notes now.

Reply:
ah, very good point.  We've done that in Android; haven't done that here yet because the Frech translation was an outside contribution.